test/03_device-state.ts

```diff
@@ -49,17 +50,11 @@ mockery.registerMock('../src/lib/config', configMock);
 mockery.registerMock('../src/lib/device-online-state', stateMock);
 
 const waitFor = async (fn: () => boolean, timeout: number = 10000) => {
```

Comment:
Can we get rid of this one altogether and just use the shared one directly? Ideally with its defaults?

Reply:
Sure

Reply:
Blind me didn't see realize that the default values gave the same 10sec timeout...